pcl_detector_node:
  ros__parameters:
    leaf_size: 0.1
    detector: "dbscan" # one of "dbscan", "euclidean"
    
    dbscan:
      epsilon: 2.0
      min_points: 20

    euclidean:
      cluster_tolerance: 1.0
      min_points: 25